### ISSUE TITLE
[Bug] Disk usage over 100%

### ISSUE THREAD

**Issue by enrico1203** [MauriceNino/dashdot#514]
### Description of the bug

Real disk space avaliable: 
![image](https://user-images.githubusercontent.com/61981530/210274460-b559939e-657a-4839-a8da-47a977a14030.png)
what i see on dash.:
![Immagine 2023-01-02 210653](https://user-images.githubusercontent.com/61981530/210274485-31626fbe-b864-4ff5-b55e-a280a41edd6a.png)

### How to reproduce

i'm using docker compose. 

```yml
version: '3.5'

services:
  dash:
    image: mauricenino/dashdot:latest
    restart: unless-stopped
    privileged: true
    environment:
      DASHDOT_ENABLE_CPU_TEMPS: 'true'
      DASHDOT_FS_VIRTUAL_MOUNTS: 'db-ita-crypt:/'
      DASHDOT_ENABLE_STORAGE_SPLIT_VIEW: 'true'
    volumes:
      - /:/mnt/host:ro
      - /proc/1/ns/net:/mnt/host_ns_net:ro
    ports:
      - '80:3001'
```

### Info output of dashdot cli

```shell
docker exec dash-dash-1 yarn cli info
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.12.1
Dash: 4.6.0

Cwd: /app
Hash: 68a114ef96f399acb69125364a744840b30c16d9
In Docker: true
In Podman: false
In Docker (env): true
Done in 0.20s.
```

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hello there, thanks for creating this issue. Could you please provide the output of the following command and I will look into it:

```
docker exec CONTAINER yarn cli raw-data --storage
```

**Comment by enrico1203** [comment]
done when on dash says: 101,8%

yarn run v1.22.19
$ node dist/apps/cli/main.js raw-data --storage
Disk Layout: [
  {
    device: '/dev/nvme0n1',
    type: 'NVMe',
    name: 'SAMSUNG MZVL2512HCJQ-00B00              ',
    vendor: 'Samsung',
    size: 512110190592,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: 'S675NX0T349018',
    interfaceType: 'PCIe',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/nvme1n1',
    type: 'NVMe',
    name: 'SAMSUNG MZVL2512HCJQ-00B00              ',
    vendor: 'Samsung',
    size: 512110190592,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: 'S675NX0T551801',
    interfaceType: 'PCIe',
    smartStatus: 'unknown',
    temperature: null
  }
]
FS Size: [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 467645579264,
    used: 260685463552,
    available: 183129796608,
    use: 58.74,
    mount: '/',
    rw: false
  },
  {
    fs: '/dev/md3',
    type: 'ext4',
    size: 467645579264,
    used: 260685463552,
    available: 183129796608,
    use: 58.74,
    mount: '/mnt/host',
    rw: true
  },
  {
    fs: '/dev/md2',
    type: 'ext3',
    size: 1036726272,
    used: 126164992,
    available: 856981504,
    use: 12.83,
    mount: '/mnt/host/boot',
    rw: true
  },
  {
    fs: '/dev/md0',
    type: 'vfat',
    size: 268087296,
    used: 176128,
    available: 267911168,
    use: 0.07,
    mount: '/mnt/host/boot/efi',
    rw: true
  },
  {
    fs: 'db-ita-crypt:/',
    type: 'fuse.rclone',
    size: 1845410008137728,
    used: 1374804697473024,
    available: 470605310664704,
    use: 74.5,
    mount: '/mnt/host/media',
    rw: false
  }
]
Block Devices: [
  {
    name: 'nvme0n1',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 512110190592,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'SAMSUNG MZVL2512HCJQ-00B00',
    serial: 'S675NX0T349018      ',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme1n1',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 512110190592,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'SAMSUNG MZVL2512HCJQ-00B00',
    serial: 'S675NX0T551801      ',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme0n1p1',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 268435456,
    physical: '',
    uuid: '8df5997d-5c07-5336-7809-e1c1d275f2fe',
    label: 'archlinux-latest-64-minimal:0',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p2',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 34359738368,
    physical: '',
    uuid: '0a811c0a-841f-c9d2-8a5c-4f39e3c91f82',
    label: 'archlinux-latest-64-minimal:1',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p3',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 1073741824,
    physical: '',
    uuid: 'cf2c6311-172d-a71c-0b49-2b2deb8a7be5',
    label: 'archlinux-latest-64-minimal:2',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p4',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 476406160896,
    physical: '',
    uuid: '7b8fc46e-10f1-612a-84a6-e6411f4dcaca',
    label: 'archlinux-latest-64-minimal:3',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme1n1p1',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 268435456,
    physical: '',
    uuid: '8df5997d-5c07-5336-7809-e1c1d275f2fe',
    label: 'archlinux-latest-64-minimal:0',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p2',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 34359738368,
    physical: '',
    uuid: '0a811c0a-841f-c9d2-8a5c-4f39e3c91f82',
    label: 'archlinux-latest-64-minimal:1',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p3',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 1073741824,
    physical: '',
    uuid: 'cf2c6311-172d-a71c-0b49-2b2deb8a7be5',
    label: 'archlinux-latest-64-minimal:2',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p4',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 476406160896,
    physical: '',
    uuid: '7b8fc46e-10f1-612a-84a6-e6411f4dcaca',
    label: 'archlinux-latest-64-minimal:3',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'md0',
    type: 'raid1',
    fsType: 'vfat',
    mount: '/mnt/host/boot/efi',
    size: 268369920,
    physical: '',
    uuid: 'F539-5EE9',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md1',
    type: 'raid1',
    fsType: 'swap',
    mount: '[SWAP]',
    size: 34325135360,
    physical: '',
    uuid: '3cd6a192-5778-48af-bfb1-8e30da09122e',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md2',
    type: 'raid1',
    fsType: 'ext3',
    mount: '/mnt/host/boot',
    size: 1071644672,
    physical: '',
    uuid: '53300eed-6ccb-4ad7-878f-9359a47fddac',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md3',
    type: 'raid1',
    fsType: 'ext4',
    mount: '/mnt/host',
    size: 476270886912,
    physical: '',
    uuid: '826abd28-f0af-4b95-8e27-34ce300b1304',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  }
]
Done in 0.16s.
[root@archlinux-latest-64-minimal ~]#

**Comment by enrico1203** [comment]
Now is at 126% 

$ node dist/apps/cli/main.js raw-data --storage
Disk Layout: [
  {
    device: '/dev/nvme0n1',
    type: 'NVMe',
    name: 'SAMSUNG MZVL2512HCJQ-00B00              ',
    vendor: 'Samsung',
    size: 512110190592,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: 'S675NX0T349018',
    interfaceType: 'PCIe',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/nvme1n1',
    type: 'NVMe',
    name: 'SAMSUNG MZVL2512HCJQ-00B00              ',
    vendor: 'Samsung',
    size: 512110190592,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: 'S675NX0T551801',
    interfaceType: 'PCIe',
    smartStatus: 'unknown',
    temperature: null
  }
]
FS Size: [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 467645579264,
    used: 322657038336,
    available: 121158221824,
    use: 72.7,
    mount: '/',
    rw: false
  },
  {
    fs: '/dev/md3',
    type: 'ext4',
    size: 467645579264,
    used: 322657038336,
    available: 121158221824,
    use: 72.7,
    mount: '/mnt/host',
    rw: true
  },
  {
    fs: '/dev/md2',
    type: 'ext3',
    size: 1036726272,
    used: 126164992,
    available: 856981504,
    use: 12.83,
    mount: '/mnt/host/boot',
    rw: true
  },
  {
    fs: '/dev/md0',
    type: 'vfat',
    size: 268087296,
    used: 176128,
    available: 267911168,
    use: 0.07,
    mount: '/mnt/host/boot/efi',
    rw: true
  },
  {
    fs: 'db-ita-crypt:/',
    type: 'fuse.rclone',
    size: 1845410008137728,
    used: 1374804697473024,
    available: 470605310664704,
    use: 74.5,
    mount: '/mnt/host/media',
    rw: false
  }
]
Block Devices: [
  {
    name: 'nvme0n1',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 512110190592,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'SAMSUNG MZVL2512HCJQ-00B00',
    serial: 'S675NX0T349018      ',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme1n1',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 512110190592,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'SAMSUNG MZVL2512HCJQ-00B00',
    serial: 'S675NX0T551801      ',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme0n1p1',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 268435456,
    physical: '',
    uuid: '8df5997d-5c07-5336-7809-e1c1d275f2fe',
    label: 'archlinux-latest-64-minimal:0',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p2',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 34359738368,
    physical: '',
    uuid: '0a811c0a-841f-c9d2-8a5c-4f39e3c91f82',
    label: 'archlinux-latest-64-minimal:1',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p3',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 1073741824,
    physical: '',
    uuid: 'cf2c6311-172d-a71c-0b49-2b2deb8a7be5',
    label: 'archlinux-latest-64-minimal:2',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme0n1p4',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 476406160896,
    physical: '',
    uuid: '7b8fc46e-10f1-612a-84a6-e6411f4dcaca',
    label: 'archlinux-latest-64-minimal:3',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme0n1'
  },
  {
    name: 'nvme1n1p1',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 268435456,
    physical: '',
    uuid: '8df5997d-5c07-5336-7809-e1c1d275f2fe',
    label: 'archlinux-latest-64-minimal:0',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p2',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 34359738368,
    physical: '',
    uuid: '0a811c0a-841f-c9d2-8a5c-4f39e3c91f82',
    label: 'archlinux-latest-64-minimal:1',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p3',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 1073741824,
    physical: '',
    uuid: 'cf2c6311-172d-a71c-0b49-2b2deb8a7be5',
    label: 'archlinux-latest-64-minimal:2',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'nvme1n1p4',
    type: 'part',
    fsType: 'linux_raid_member',
    mount: '',
    size: 476406160896,
    physical: '',
    uuid: '7b8fc46e-10f1-612a-84a6-e6411f4dcaca',
    label: 'archlinux-latest-64-minimal:3',
    model: '',
    serial: '',
    removable: false,
    protocol: 'nvme',
    group: '',
    device: '/dev/nvme1n1'
  },
  {
    name: 'md0',
    type: 'raid1',
    fsType: 'vfat',
    mount: '/mnt/host/boot/efi',
    size: 268369920,
    physical: '',
    uuid: 'F539-5EE9',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md1',
    type: 'raid1',
    fsType: 'swap',
    mount: '[SWAP]',
    size: 34325135360,
    physical: '',
    uuid: '3cd6a192-5778-48af-bfb1-8e30da09122e',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md2',
    type: 'raid1',
    fsType: 'ext3',
    mount: '/mnt/host/boot',
    size: 1071644672,
    physical: '',
    uuid: '53300eed-6ccb-4ad7-878f-9359a47fddac',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'md3',
    type: 'raid1',
    fsType: 'ext4',
    mount: '/mnt/host',
    size: 476270886912,
    physical: '',
    uuid: '826abd28-f0af-4b95-8e27-34ce300b1304',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  }
]
Done in 0.16s.
